volumina/view3d/glview.py

```diff
@@ -204,7 +204,7 @@ class GLViewMock(QLabel):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.setText(
-            "3D widget disabled via $HOME/.voluminarc or environment variable" "<pre>FEATURES_USE_OPENGL_WIDGET</pre>"
```

Comment:
should we also specify which of the `.voluminarc` options handles disabling the widget?

Reply:
hm, since the default has it on, and this message will only be shown if it is off, I think it should be fine.
One the other hand, doesn't hurt to be more explicit.